vm/proxyapp/proxyappclient.go

```diff
@@ -185,6 +197,35 @@ func runProxyApp(params *proxyAppParams, cmd string) (*ProxyApp, error) {
 	}, nil
 }
 
+func (proxy *ProxyApp) doLogPooling(writer io.Writer) {
+	proxy.stopLogPooling = make(chan bool, 1)
+	go func() {
+		for {
+			var reply proxyrpc.PoolLogsReply
+			call := proxy.Go(
+				"ProxyVM.PoolLogs",
+				&proxyrpc.PoolLogsParam{},
+				&reply,
+				nil,
+			)
+			select {
+			case <-proxy.stopLogPooling:
+				return
+			case c := <-call.Done:
+				if c.Error != nil {
+					// possible errors here are:
+					// "unexpected EOF"
+					// "read tcp 127.0.0.1:56886->127.0.0.1:34603: use of closed network connection"
+					log.Logf(0, "error pooling ProxyApp logs: %v", c.Error)
+					return
+				}
+				writer.Write([]byte(fmt.Sprintf("ProxyAppLog severity %v: %v",
```

Comment:
Severity is usually used to limit too verbose output (e.g. debug mode with lots of verbose output vs normal mode with less output), or to terminate the application of fatal errors.
Here it does not affect anything (to ignore a debug message a user still needs to read the line, which defeats the goal of not bothering the user with this info).
We can either accept `func(verbosity int, data string)` as `params.LogOutput`, or add a `V` function to our `pkg/log`:
https://pkg.go.dev/github.com/golang/glog#V
and check the global verbosity level here and print only subset of messages.

Reply:
Added V(),